packages/config/test/options.test.js

```diff
@@ -112,11 +112,21 @@ describe('config: options', () => {
     expect(pageTransition.appear).toEqual(true)
   })
 
-  test('should return 404.html as default generate.fallback', () => {
+  test('should return 200.html as default generate.fallback', () => {
```

Comment:
fallback is also tested in `generator.gen.test.js` -> `should write fallback html after generate`